modules/gitops-deploy-resources-different-namespaces.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * argo-cd-custom-resource-properties.adoc
+
+:_content-type: PROCEDURE
+[id="deploy-resources-different-namespaces_{context}"]
+= Deploy resources to a different namespace
+
+[role="_abstract"]
+To allow the Argo CD to manage resources in other namespaces apart from where it is installed, configure the target namespace with a label "argocd.argoproj.io/managed-by".
+
+.Procedure
+. Run the following command to configure the namespace.
```

Comment:
```suggestion
. Configure the namespace by running the following command:
```
OR
```suggestion
. Configure the namespace:
```
- For required steps, use the imperative verb. 
- In situations like this, you don't need to mention the command.